### ISSUE TITLE
Failure alerts on scenario-testing

### ISSUE THREAD

**Issue by mbifeld** [Azure/InnovationEngine#91]
Alerts on testing failures based on the ms.author / author tag in the metadata yml at the top of docs.

![Image](https://github.com/Azure/InnovationEngine/assets/55719566/3bb26260-f36f-4a83-bc81-38f82a91e73b)

**Comment by mbifeld** [comment]
Depends on [#170](https://github.com/Azure/InnovationEngine/issues/170)

**Comment by rgardler-msft** [comment]
These are Azure specific meta-data fields and this handling them is out of scope for IE. Closing as Won't Fix.
